src/Utility/TypeExtensions.cs

```diff
@@ -153,9 +153,9 @@ internal static RpcException ToRpcException(this Exception exception)
         {
```

Comment:
Did you change anything on this file beside the order or parameters?

Reply:
No, it was a nitpick to make the order of parameters the same as specified in the protobuf.